scala_util/pom.xml

```diff
@@ -66,13 +66,13 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.binary.version}</artifactId>
-            <version>2.2.4</version>
+            <version>${scalatest.version}</version>
```

Comment:
Since many modules have this dependency, it should be added to <dependencyManagement> in the topmost pom.xml

Reply:
moved version to the parent pom